src/index.js

```diff
@@ -26,13 +26,29 @@ const getRootNode =
     ? (element) => element?.getRootNode?.()
     : (element) => element?.ownerDocument;
 
+/**
+ * Determines if a node is inert or in an inert ancestor.
+ * @param {Element} node
+ * @param {boolean} [lookUp] If true and `node` is not inert, looks up at ancestors to
+ *  see if any of them are inert. If false, only `node` itself is considered.
+ * @returns {boolean} True if inert itself or by way of being in an inert ancestor.
+ *  False if `node` is falsy.
+ */
+const isInert = function (node, lookUp = true) {
+  return !!(node?.inert || (lookUp && node && isInert(node.parentNode))); // recursive
```

Comment:
Why do you need to lookup with JavaScript instead of matching with CSS match: 

With parent check:
is not inert: `node.matches(':not([inert]), :not([inert] *)')`
is inert:        `node.matches('[inert], :is([inert] *)')`

Reply:
Good point, but unfortunately we can't call matches on what I think are the shadow roots. Lots of shadow-related tests fail because of `matches is not a function` errors when I use your suggestion. What I have right now works in all cases.